testsuite/elytron/src/test/java/org/wildfly/test/security/common/elytron/Path.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2017, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.wildfly.test.security.common.elytron;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static org.jboss.as.test.shared.CliUtils.escapePath;
+
+/**
+ * Helper class for adding "path" and "relative-to" attributes into CLI commands.
+ *
+ * @author Josef Cacek
+ */
+public class Path implements CliFragment {
```

Comment:
I think we should rename this to something like `CliPath` only for the fact that some tests that end up using this *may* also want to use a `java.nio.file.Path`.